src/Illuminate/Foundation/Http/Exceptions/MalformedUrlException.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Illuminate\Foundation\Http\Exceptions;
+
+use Symfony\Component\HttpKernel\Exception\HttpException;
+
+class MalformedUrlException extends HttpException
+{
+    /**
+     * Create a new exception instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct(400, 'Malformed URL.');
```

Comment:
Would you want to use the [`\Symfony\Component\HttpFoundation\Response` status code constants](https://github.com/symfony/symfony/blob/04551ad54245e0f48206eb05a582996b419c8c1e/src/Symfony/Component/HttpFoundation/Response.php#L24-L86)?
```suggestion
        parent::__construct(Response::HTTP_BAD_REQUEST, 'Malformed URL.');
```

Reply:
I will leave this decision up to the maintainers, it doesn't really matter to me.